src/sentry/static/sentry/app/components/eventOrGroupHeader.jsx

```diff
@@ -43,15 +44,22 @@ class EventOrGroupHeader extends React.Component {
     const basePath = `/organizations/${orgId}/issues/`;
 
     if (includeLink) {
+      const locationWithProject = {...browserHistory.getCurrentLocation()};
+      const query =
+        locationWithProject.query.project !== undefined
+          ? {
+              query: this.props.query,
+            }
+          : {query: this.props.query, _allp: 1};
```

Comment:
1) We should add a comment about why we're doing this.
2) Is the assumption that users without global views (multiple projects) will always have a  project defined? Is it possible there's a race condition where it's not?

Reply:
I don't think there is a race condition because the project is loaded before issues are.